monitoring-dashboard/components/org.wso2.micro.integrator.dashboard.web/web-app/src/pages/Users.js

```diff
@@ -41,25 +38,13 @@ export default function Users() {
         tableOrderBy: 'name'
     });
 
-    const [users, setUsers] = React.useState(null);
     const [error, setError] = React.useState(null);
     const classes = useStyles();
     const globalGroupId = useSelector(state => state.groupId);
     const dataSet = useSelector(state => state.data);
 
-    const retrieveUsers = () => {
-        HTTPClient.getUsers(globalGroupId).then(response => {
-            response.data.map(data => data.details = JSON.parse(data.details));
-            setUsers(response.data)
-        }).catch(error => {
-            if (error.response.status === 500) {
-                setError(error.response.data.message)
-            }
-        })
```

Comment:
this was discussed previously. Intentionally removed this because applying it in the current context caused unwanted error alerts which make the user confusing